### PR TITLE
Fix joey

### PR DESCRIPTION
**Pull request by KnightVision90**
Fixes and enhancements for commands tested for #288 including:

- Fix blacklist to allow removal of guilds and users.
- Make embed color command match file name
- Enforce hex code (3 and 6 length) for embed color (https://github.com/Spudnik-Group/Spudnik-Docs/pull/3)
- Ensure command/group exists for enable command.

### CODE REVIEW

File: src/lib/types/settings/ClientSettings.ts [hunk 1]
```diff
@@ -4,8 +4,8 @@ import { T } from './Shared';
 export namespace ClientSettings {
 
 	export namespace Blacklist {
-		export const Guilds = T<string[]>('blacklist.guilds');
```
Comment (develogix) [inline src/lib/types/settings/ClientSettings.ts:7]: Keep this schema; change in `src/commands/bot-owner/blacklist.ts` to match to prior commit of this file.

File: src/commands/bot-owner/blacklist.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ export default class BlacklistCommand extends Command {
 
 	public constructor(store: CommandStore, file: string[], directory: string) {
 		super(store, file, directory, {
-			description: (language: any) => language.get('COMMAND_BLACKLIST_DESCRIPTION'),
```
Comment (develogix) [inline src/commands/bot-owner/blacklist.ts:23]: Why was this changed?

Comment (KnightVision90) [reply src/commands/bot-owner/blacklist.ts:23]: This was not actually being called and was not returning a proper description.
image

**Comment by develogix** [comment]
PR denied
